app/pages/HomePage/components/BreakingNews/BreakingNews.module.scss

```diff
@@ -0,0 +1,61 @@
+//.breakingNews {
+//  height: 220px;
+//  color: #000000;
+//  padding: 1rem 1.75rem;
+//}
+//
+//.title {
+//  color: black;
+//  font-family: 'Open Sans', serif;
+//  font-style: normal;
+//  font-weight: 700;
+//  font-size: 26px;
+//  line-height: 35px;
+//  width: 100%;
+//}
+//
+
+
+
+
+
+
+.section {
+  color: #202020;
+  margin-top: 100px;
+  position: relative;
+  @media screen and ( max-width: 600px ) {
+    margin-top: 50px;
+  }
+}
+
+.container {
+  //background-color: #E5E5E5;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: (1280px - 120px); // Max width of 1280px including side margins of 60px
+  width: 90%;
+  border-radius: 8px;
+  border: solid 1px transparent;
+  box-shadow: 0 6px 16px rgb(0 0 0 / 20%);
+}
+
+.title {
+  font-weight: 600;
+  font-size: 42px;
+  @media screen and ( max-width: 767px ) {
```

Comment:
Let's use the constants for the breakpoints defined in https://github.com/ShelterTechSF/askdarcel-web/blob/5604e3f3618a2a7c2c5107b7d20bdf2da9c451c4/app/styles/utils/_layoututils.scss#L4-L17 rather than hardcoding magic numbers in this file.